TheWarWithin/MageFire.lua

```diff
@@ -120,28 +121,28 @@ spec:RegisterTalents( {
     wildfire                  = { 101001, 383489, 1 }, -- Your critical strike damage is increased by 3%. When you activate Combustion, you gain 2% additional critical strike damage, and up to 4 nearby allies gain 1% critical strike for 10 sec.
 
     -- Sunfury
-    burden_of_power           = {  94644, 451035, 1 }, -- Conjuring a Spellfire Sphere increases the damage of your next Pyroblast by 15% or your next Flamestrike by 60%.
+    combustion                = { 100995, 190319, 1 }, -- Engulfs you in flames for $d, increasing your spells' critical strike chance by $s1%$?a383967[ and granting you Mastery equal to $s3% of your Critical Strike stat][]. Castable while casting other spells.$?a383489[; When you activate Combustion, you gain $383489s3% Critical Strike damage, and up to $383493I nearby allies gain $383489s4% Critical Strike for $383493d.][]
```

Comment:
Is this a Sunfury talent or does it belong up above?

Reply:
Looks like a bad copy/paste somehow, it already exists above.